doc/source/methods/TreeSHAP.ipynb

```diff
@@ -0,0 +1,586 @@
+{
```

Comment:
<p>I would spend some time clarifying what exactly is going on, happy to help. Here are some high level ideas:</p><ul><li>Be explicit that the following section is about calculating contribution wrt <strong>one</strong> reference value (on one tree) and that these are in the end averaged over</li><li>Rewrite the first sentence of feature perturbation to make it shorter and clearer, connect with how "missing features" translate to "replaced by values from the reference instance"</li><li>Explicitly define the positive/negative contribution and how these relate to splits in the tree</li><li>Spell out that the diagram is a decision tree splitting on different features at different nodes. Use this to clear up about what is meant by not being able to estimate contributions for F2, F4 and how v6 and v10 relate. Could spend a whole paragraph just setting up the scene</li><li>Maybe defer the figure after a few paragraphs to enhance the narrative flow</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/206/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>